src/containers/Tenant/Diagnostics/Healthcheck/Healthcheck.tsx

```diff
@@ -4,7 +4,12 @@ import cn from 'bem-cn-lite';
 
 import {Loader} from '@gravity-ui/uikit';
 
-import {getHealthcheckInfo} from '../../../../store/reducers/healthcheckInfo';
+import {SelfCheckResult} from '../../../../types/api/healthcheck';
+import {
+    getHealthcheckInfo,
+    selectInvertedIssuesConsequenceTrees,
+    selectIssueConsequenceById,
+} from '../../../../store/reducers/healthcheckInfo';
 import {useAutofetcher} from '../../../../utils/hooks';
 
 import {IssuesList} from './IssuesList';
```

Comment:
as for the naming, i'd go with something like

```
Healthcheck/
  Preview
  Details // currently IssuesList 
  IssuesTree // currently IssuesViewer
```

`IssuePreview` is clashing with `Preview`. So there may be point in moving it inside Preview, like `IssueRow` is inside `IssuesViewer` now. On the other hand, if `IssuePreview` remains a separate component, we should make an `IssueRow` one as well, to maintain structural consistency.

In case they are separate components, they could be named something like `PreviewItem` and `IssuesTreeItem`